frontend/src/views/Lobby.tsx

```diff
@@ -46,6 +52,17 @@ function LobbyPage() {
     console.log(user);
   };
 
+  const handleClick = () => {
+    const request = { roomId: currentRoomId, initiator: currentUser as User };
+    startGame(request)
+      .then(() => {
```

Comment:
I think request can be passed as a parameter to startGame instead of in the request body now that you changed it to use a path variable